Source/AliveLibAE/DDraw.cpp

```diff
@@ -625,7 +625,7 @@ EXPORT signed int CC DD_Enable_4F0380(HWND /*hwnd*/, int width, int height, int
         ::ReleaseDC(nullptr, dc);
 
         const LONG oldWinStyle = ::GetWindowLongA(sDD_hWnd_BBC3B0, GWL_STYLE);
-#if BEHAVIOUR_CHANGE_FORCE_WINDOW_MODE
+#ifdef BEHAVIOUR_CHANGE_FORCE_WINDOW_MODE
```

Comment:
I'm not sure this is correct - BEHAVIOUR_CHANGE_FORCE_WINDOW_MODE is always defined by cmake as 

#define BEHAVIOUR_CHANGE_FORCE_WINDOW_MODE 0
or
#define BEHAVIOUR_CHANGE_FORCE_WINDOW_MODE 1

so now it appears that this block will always be compiled in no matter the macro value

Reply:
Yes, my bad. I was playing around with these to figure out the invalid PCH issue and forgot to turn them back.